project/MimaExcludes.scala

```diff
@@ -1476,6 +1476,12 @@ object MimaExcludes {
       ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("org.apache.spark.ml.param.shared.HasLoss.org$apache$spark$ml$param$shared$HasLoss$_setter_$loss_="),
       ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("org.apache.spark.ml.param.shared.HasLoss.getLoss"),
       ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("org.apache.spark.ml.param.shared.HasLoss.loss")
+    ) ++ Seq(
+      // [SPARK-26311][CORE]New feature: apply custom log URL pattern for executor log URLs
```

Comment:
Where is this being added? This looks like the `v20excludes` variable. These excludes should be added at the very top of this file.

Reply:
My bad. I missed variables and thought there's one huge list containing everything. Will move to `v30excludes`.